src/core/storage/sframe_interface/unity_sarray.cpp

```diff
@@ -1764,6 +1764,19 @@ std::shared_ptr<unity_sarray_base> unity_sarray::tail(size_t nrows) {
   return copy_range(start, 1, end);
 }
 
+std::shared_ptr<unity_sarray_base> unity_sarray::make_uniform_int_array(size_t size, size_t max_int) {
+  int seed_hash = 12; // XXX: how to set this properly?
```

Comment:
uint64_t seed = random::pure_random_seed(); 

This is best here because group-by doesn't guarantee order because of the multithreaded aspect of this.  If we want deterministic order, we'll need to do a sort.

Reply:
Done. 

Yeah, I was hoping we could expose a `seed` parameter for `shuffle` in the Python layer but that's not going to be possible.